network-api/networkapi/wagtailpages/pagemodels/customblocks/image_grid.py

```diff
@@ -18,6 +33,9 @@ class ImageGrid(blocks.StructBlock):
         help_text='If left checked, the image will be cropped to be square.'
     )
 
+    class Meta:
+        value_class = ImageGridAltText
```

Comment:
I keep forgetting this is a thing. It's kind of deep wagtail/django magic, but a nice solution here.

Reply:
Thanks!